[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  postgresql-k8s/0 [idle] active: 
  postgresql-k8s/1 [idle] active: Primary
  postgresql-k8s/2 [idle] active: Primary
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  postgresql-k8s/0 [idle] active: 
  postgresql-k8s/1 [idle] active: 
  postgresql-k8s/2 [idle] active: Primary
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  postgresql-k8s/0 [idle] active: 
  postgresql-k8s/1 [idle] active: 
  postgresql-k8s/2 [idle] active: Primary
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  postgresql-k8s/0 [executing] active: 
  postgresql-k8s/1 [executing] active: 
  postgresql-k8s/2 [executing] waiting: Awaiting restart operation
[32mINFO    [0m pytest_operator.plugin:plugin.py:903 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.5    unsupported  01:44:43Z

App                       Version  Status  Scale  Charm                     Channel      Rev  Address         Exposed  Message
postgresql-k8s            14.13    active      3  postgresql-k8s                           0  10.152.183.129  no       
self-signed-certificates           active      1  self-signed-certificates  latest/edge  197  10.152.183.65   no       

Unit                         Workload  Agent      Address       Ports  Message
postgresql-k8s/0             active    executing  10.1.236.140         (get-password) 
postgresql-k8s/1*            active    idle       10.1.236.141         
postgresql-k8s/2             active    idle       10.1.236.142         Primary
self-signed-certificates/0*  active    idle       10.1.236.139         

[32mINFO    [0m pytest_operator.plugin:plugin.py:909 Juju error logs:

unit-self-signed-certificates-0: 01:42:49 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#postgresql-k8s": settings 00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#postgresql-k8s not found (not found)
unit-postgresql-k8s-2: 01:42:57 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#self-signed-certificates": settings 00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#self-signed-certificates not found (not found)
unit-postgresql-k8s-0: 01:42:58 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#self-signed-certificates": settings 00f918ca-4314-4f32-803b-c4c4e0a8ef2b:r#3#self-signed-certificates not found (not found)

[32mINFO    [0m pytest_operator.plugin:plugin.py:991 Forgetting model main...